content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- ***Added*** the new [Workload Entry](/docs/reference/config/networking/workload-entry/) resource. This allows easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** configuration for gateway topology. This addresses providing correct [X-Forwarded-For headers](https://github.com/istio/istio/issues/7679) and X-Forwarded-Client-Cert headers based on gateway deployment topology .
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis/).
+- ***Added*** support for using `appProtocol` to select the [protocol for a port](/docs/ops/configuration/traffic-management/protocol-selection/) introduced in Kubernetes 1.18.
+- ***Changed*** gateway SDS to be enabled by default. File mounted gateway continues to be available to help users to transition to secure gateway SDS.
```

Comment:
Do you mean Gateway the kind?  If so it should have a capital G

Reply:
Done